tools/automate_bindings.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python
+
+""" Generate bindings of given Wt's class and all its dependencies.
+
+Dependencies are all other classes from interface of methods
+of given class (types of arguments and return values).
+"""
+
+import argparse
+import os
+
+from pygccxml import declarations
+from pygccxml import parser
+from pygccxml import utils
+
+TYPE_FROM_LUA_FUNCS = {
+    'int' : 'lua_tointeger',
+    'bool' : 'lua_toboolean',
+}
+
+TYPE_TO_LUA_FUNCS = {
+    'int' : 'lua_pushinteger',
+    'bool' : 'lua_pushboolean',
+}
+
+INCLUDES_TEMPLATE = r'''
+#include "boost-xtime.hpp"
+#include <Wt/%s>
+
+#include "globals.hpp"
+
+'''
+
+MODULE_FUNC_TEMPLATE = r'''
+void luawt_%(module_name)s(lua_State* L) {
+    const char* base = luawt_typeToStr<%(base)s>();
+    assert(base);
+    DECLARE_CLASS(
+        %(module_name)s,
+        L,
+        wrap<luawt_%(module_name)s_make>::func,
+        0,
+        luawt_%(module_name)s_methods,
+        %(base)s
+    );
+}
+'''
+
+RETURN_CALLS_TEMPLATE = r'''
+    %s(L, result);
+    return 1;
+'''
```

Comment:
`INCLUDES_TEMPLATE`, `MODULE_FUNC_TEMPLATE` and `RETURN_CALLS_TEMPLATE` are used only once. Can you inline them?